examples/model_compress/combined_compression.py

```diff
@@ -0,0 +1,309 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
looks good, maybe it is better to write this example using Jupyter Notebook, let's leave this task to another one, maybe @linbinskn :)

Reply:
better to rename this file to "end2end_compression.py"

Reply:
I think our sample is a bit unfriendly to users（with too many input parameters). Shall we add a README to this example like AMC(https://github.com/microsoft/nni/blob/25dea8f5528be7f3f74ca2c2fa42f528f6cdd4eb/examples/model_compress/pruning/amc/README.md)?